src/api/base.js

```diff
@@ -1,5 +1,40 @@
 import axios from "axios";
+import { useDispatch, useSelector } from "react-redux";
+import jwt_decode from "jwt-decode";
+
+import { actions } from "store/features/userSlice";
+import { getTimestamp } from "utils/getTimestamp";
+
+import authApi from "./auth";
 
 export const apiClient = axios.create({
   baseURL: process.env.REACT_APP_BACKEND_HOST,
 });
+
+apiClient.interceptors.request.use(
+  function (response) {
+    return response;
+  },
+  async (error) => {
+    const { decodedToken, refreshToken } = useSelector((state) => state.user);
+    const dispatch = useDispatch();
+    if (decodedToken.payload.exp < getTimestamp()) {
```

Comment:
Declare please an additional constant for `getTimestamp()`. We shouldn't call any functions inside comparison